geospatial/include/gz/common/geospatial/Dem.hh

```diff
@@ -52,12 +60,12 @@ namespace gz
       /// \brief Indicate that this is a non Earth DEM.
       /// \param[in] _isNonEarthDem Should be true if this is a
       /// non earth DEM, otherwise false.
```

Comment:
```suggestion
      /// \brief Indicate that the coordinate system of the DEM is unknown.
      /// \param[in] _isUnknownDem True if the coordinate system
      /// is unknown, otherwise false.
```

Reply:
Done https://github.com/gazebosim/gz-common/pull/377/commits/e42c08d9e12ccf1902a20f930322f79a89d03238